paasta_tools/utils.py

```diff
@@ -771,14 +772,14 @@ def stringify_constraints(uscs: Optional[List[UnstringifiedConstraint]]) -> List
 @time_cache(ttl=60)
 def validate_service_instance(service: str, instance: str, cluster: str, soa_dir: str) -> str:
     for instance_type in INSTANCE_TYPES:
-        services = get_services_for_cluster(cluster=cluster, instance_type=instance_type, soa_dir=soa_dir)
-        if (service, instance) in services:
+        sis = get_services_for_cluster(cluster=cluster, instance_type=instance_type, soa_dir=soa_dir)
+        if (service, instance) in sis:
             return instance_type
     else:
+        suggestions = suggest_possibilities(word=instance, possibilities=[si[1] for si in sis])
```

Comment:
These will only be for the last instance type.

Reply:
Hah. Hmm. Who would ever need anything other than suggestions for tron? :)